certbot/tests/util_test.py

```diff
@@ -470,79 +470,6 @@ def test_wildcard(self):
         self.assertTrue(self._call(self.wildcard.encode()))
 
 
-class OsInfoTest(unittest.TestCase):
```

Comment:
I think we may want to add some of these tests back depending on your response to my suggestions.